drivers/adc/ad7779/ad7779.c

```diff
@@ -1554,6 +1554,8 @@ int32_t ad7779_init(ad7779_dev **device,
 			ret |= ad7779_set_state(dev, (ad7779_ch)i, dev->state[i]);
 	}
 
+	no_os_mdelay(50);
```

Comment:
is this delay value a trial-and-error value or does it come from the datasheet ?

Reply:
I do not see any value specific to this in the datasheet. It is a trial and error value that has been tested with two STM32 boards interfaced with the AD7770 chip